modules/installation-user-infra-machines-advanced-customizing-iso-or-pxe.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-iso-or-pxe_{context}"]
+= Customizing a live {op-system} ISO or PXE install
+You can use the live ISO image or PXE environment to install {op-system} by injecting an Ignition config file directly into the image. This creates a customized image which you can then use to boot a live system that is provisioned with the given Ignition config.
+
+For an ISO image, the mechanism to do this is the `coreos-installer iso customize` subcommand which modifies the `.iso` file with your configuration. Similarly, the mechanism for a PXE environment is the `coreos-installer pxe customize` subcommand which creates a new initramfs file that combines your configuration with the PXE image.
+
+In addition to embedding Ignition configs directly, the `customize` subcommand is a general-purpose tool that can take multiple other arguments and create Ignition configs from them to embed in the live system. Some common tasks include:
+
+* Modifying a pointer config to inject directly into the image.
```

Comment:
We don't modify the pointer config, and docs don't normally call it a pointer config.